src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1643,10 +1643,16 @@ because xbuild doesn't support framework reference assemblies.
       LibraryProjectImportsDirectoryName="$(_LibraryProjectImportsDirectoryName)">
     <Output TaskParameter="ExtraPackages" PropertyName="AaptExtraPackages" />
   </GetExtraPackages>
-  
+
```

Comment:
This is the only real change.

Reply:
If this is the only "real" change, could we revert the file and apply *just* this change?

Additionally, could the `Condition="…"` value be the *first* attribute instead of the last?